assemblies/core/static/src/main/resources-filtered/spoon.sh

```diff
@@ -191,23 +191,11 @@ case `uname -s` in
 		# linux compatibility packages installed
 	    ARCH=`uname -m`
 		case $ARCH in
-			x86_64)
-				LIBPATH=$CURRENTDIR/../libswt/linux/x86_64/
-				echo "I'm sorry, this FreeBSD platform [$ARCH] is not yet supported!"
-				exit
-				;;
-
 			i[3-6]86)
 				LIBPATH=$CURRENTDIR/../libswt/linux/x86/
 				;;
 
-			ppc)
-				LIBPATH=$CURRENTDIR/../libswt/linux/ppc/
```

Comment:
In the PDI assembly we have under `libswt`:
- ./win32
- ./win64
- ./osx64
- ./linux/x86
- ./linux/x86_64

There are a few others that should be removed like `libswt/linux/ppc64/` and `libswt/solaris-x86/` above.